docs/resources/variable.md

```diff
@@ -0,0 +1,33 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "tharsis_variable Resource - terraform-provider-tharsis"
+subcategory: ""
+description: |-
+  Defines and manages a namespace variable.
+---
+
+# tharsis_variable (Resource)
+
+Defines and manages a namespace variable.
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `category` (String) Whether this variable is a Terraform or an environment variable.
+- `hcl` (Boolean) Whether this variable has an HCL value.
+- `key` (String) This variable's key (within its host namespace).
+- `namespace_path` (String) The path to this variable's host namespace.
```

Comment:
The word `host` can be moved from a few of these descriptions to avoid any confusion.